### PR TITLE
[DF] Add test for I/O of nested RVecs

### PR DESCRIPTION
**Pull request by eguiraud**

### CODE REVIEW

File: root/dataframe/test_nested_rvec_snapshot.C [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+struct TwoInts {
+   int a, b;
+};
+
+struct TriggerVecOpsDictGeneration {
+   RVec<TwoInts> _1;
+   RVec<RVec<TwoInts>> _2;
+   RVec<RVec<int>> _3;
+   RVec<RVec<RVec<int>>> _4;
+};
```
Comment (pcanal) [inline root/dataframe/test_nested_rvec_snapshot.C:14]: FYI, using _here_
```
#ifdef __ROOTCLING__
#prama link C++ class ROOT::VecOps::RVec<TwoInts>+;
...
```
would (should) have also worked.

Comment (eguiraud) [reply root/dataframe/test_nested_rvec_snapshot.C:14]: ah, did not realize that, that's better

Comment (eguiraud) [reply root/dataframe/test_nested_rvec_snapshot.C]: done

**Comment by eguiraud** [comment]
@phsft-bot build please

**Comment by eguiraud** [comment]
Rebased on top of master, which should fix the test failures (I always forget that for `roottest` the bot does not do it itself).

**Comment by eguiraud** [comment]
@bellenot fyi, seemingly random timeouts on windows

**Comment by bellenot** [comment]
> @bellenot fyi, seemingly random timeouts on windows

That's weird. And new...

**Comment by bellenot** [comment]
@eguiraud you probably missed this one: https://lcgapp-services.cern.ch/root-jenkins/job/roottest-pullrequests-build/8517/testReport/junit/projectroot.roottest.root/dataframe/roottest_root_dataframe_test_nested_rvec_snapshot/

**Pull request merged**